survey_app/build.gradle

```diff
@@ -126,13 +132,25 @@ dependencies {
         primaryImplementation(group: odkxGroupId, name: commonPackageName, version: primaryVersion)
         snapshotImplementation(group: odkxGroupId, name: commonPackageName, version: snapshotVersion)
     }
-
-    // Testing dependencies
-    testImplementation 'junit:junit:4.13.2'
-
+// Testing-only dependencies
     androidTestImplementation 'androidx.test:runner:1.5.2'
     androidTestImplementation 'androidx.test:rules:1.5.0'
+
     androidTestImplementation 'androidx.annotation:annotation:1.7.1'
+    androidTestImplementation 'androidx.test.espresso:espresso-core:3.5.1'
+    androidTestImplementation 'androidx.test.espresso:espresso-intents:3.5.1'
+    androidTestImplementation 'androidx.test.ext:junit:1.1.5'
+    androidTestImplementation 'org.mockito:mockito-core:2.19.0'
+    androidTestImplementation 'com.google.truth:truth:1.1.3'
+    androidTestImplementation 'androidx.test:core:1.5.0'
+    androidTestImplementation 'androidx.test.espresso:espresso-contrib:3.5.1'
+
+    testImplementation 'junit:junit:4.13.2'
+    testImplementation 'org.mockito:mockito-core:2.19.0'
```

Comment:
I missed this before, why are you adding the dependencies to "testImplementation" if you only need them when you run androidTest (note: tests in "androidTest" directory use androidTestImplementation dependencies, tests in "test" directory uses testImplementation dependencies). You did not add tests to the test directory so not sure why needed

Reply:
Thanks for the feedback. I'll take those out.